CWE-94-Python_darwin/747929791_majsoul_wrapper/sdk.py

```
@@ -0,0 +1,483 @@
+        if type == Operation.NoEffect:
+            print('Do nothing')
+        elif type == Operation.Chi:
+            print('Chi')
+        elif type == Operation.Peng:
+            print('Peng')
+        elif type in (Operation.MingGang, Operation.JiaGang):
+            print('Gang')
+
+
+def dumpWebSocket(filename='ws_dump.pkl'):
+    # 监听mitmproxy当前websocket，将所有报文按顺序交由handler.parse
```

Ruff (0.13.2)

Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)